monitoring/uss_qualifier/scd/executor/test_executor.py

```diff
@@ -0,0 +1,77 @@
+from monitoring.monitorlib.scd_automated_testing.scd_injection_api import InjectFlightRequest,  InjectFlightResult
+from monitoring.uss_qualifier.rid.utils import InjectionTargetConfiguration
+from monitoring.uss_qualifier.scd.data_interfaces import AutomatedTest, TestStep, FlightInjectionAttempt, \
+    InjectionTarget, FlightDeletionAttempt, KnownResponses
+from monitoring.uss_qualifier.scd.executor.executor import combine_targets
+from monitoring.uss_qualifier.scd.executor.target import TestTarget
+
+# Constants
+FirstMoverRole = "First-Mover USS"
+SecondMoverRole = "Second-Mover USS"
+
+# Test data definition
+automated_test = AutomatedTest(
+    name="Unit Test",
+    steps = [
+        TestStep(
+            name="Inject Flight 1",
+            inject_flight=FlightInjectionAttempt(
+                reference_time="2022-02-11T09:00:05.359502+00:00",
+                name="f0001",
+                test_injection=InjectFlightRequest(
+                    operational_intent=None,
+                    flight_authorisation=None
+                ),
+                known_responses=KnownResponses(
+                    acceptable_results=[InjectFlightResult.Planned],
+                    incorrect_result_details={}
+                ),
+                injection_target=InjectionTarget(uss_role=FirstMoverRole)
+            )
+        ),
+        TestStep(
+            name="Inject Flight 2",
+            inject_flight=FlightInjectionAttempt(
+                reference_time="2022-02-11T09:30:05.359502+00:00",
+                name="f0002",
+                test_injection=InjectFlightRequest(
+                    operational_intent=None,
+                    flight_authorisation=None
+                ),
+                known_responses=KnownResponses(
+                    acceptable_results=[InjectFlightResult.Planned],
+                    incorrect_result_details={}
+                ),
+                injection_target=InjectionTarget(uss_role=SecondMoverRole)
+            )
+        ),
+        TestStep(
```

Comment:
nit: I don't [think](https://github.com/interuss/dss/pull/710#discussion_r806135052) we need this step

Reply:
Since TestStep supports deleting flights as part of scenarios (even if not used yet), I think we should keep it in the tests.